File: Week 1 C/this version of Mario/Mario1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h <= 0);

    for (int row = 1; row <= h; row++)
    {
        for (int s = 0; s < h - row; s++)
        {
            printf(" ");
        }

        for (int hash = 0; hash < row; hash++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: Week 6 Python/Solution/150395760/sentimental-mario/mario.py
from cs50 import get_int #importar la funcion para usar enteros

while True:
    n = get_int("Height: ")
    # Si n está entre 1 y 8, sino romper el bloque
    if n > 0 and n < 9:
        break

for i in range(1, n + 1):
    # Imprime (espacios) + (hashtags)
    print(" " * (n - i) + "#" * i)
